src/components/modals/AddressModal.js

```diff
@@ -35,7 +35,8 @@ type OwnProps = {
   showPasteButton?: boolean,
   isFioOnly?: boolean,
   useUserFioAddressesOnly?: boolean,
-  checkAddressConnected?: boolean
+  checkAddressConnected?: boolean,
+  autoFocus?: boolean
```

Comment:
Why is this optional? Shouldn't the modal just do this by default?

If there is a good reason, then carry on - approved.